internal/corazarules/rule_match.go

```diff
@@ -130,21 +130,19 @@ func (mr MatchedRule) details(matchData types.MatchData) string {
 
 func (mr MatchedRule) matchData(matchData types.MatchData) string {
 	log := &strings.Builder{}
-	for _, matchData := range mr.MatchedDatas_ {
-		v := matchData.Variable().Name()
-		if matchData.Key() != "" {
-			v += fmt.Sprintf(":%s", matchData.Key())
-		}
-		value := matchData.Value()
-		if len(value) > 200 {
-			value = value[:200]
-		}
-		if mr.Rule_.Operator() != "" {
-			log.WriteString(fmt.Sprintf("Matched \"Operator %s matched %s at %s.",
-				"", value, v))
-		} else {
-			log.WriteString("Matched.\"")
-		}
+	v := matchData.Variable().Name()
+	if matchData.Key() != "" {
+		v += fmt.Sprintf(":%s", matchData.Key())
```

Comment:
It would be nice to remove the intermediate strings, while it's more lines I guess it's still not so bad. We're basically expanding printf into code, which should be worth it for this very hot path function

```go
log.WriteString("Matched Operator ")
log.WriteString(op)
log.WriteString(" matched ")
log.WriteString(value)
log.WriteString(" at ")
log.WriteString(matchData.Variable().Name())
if matchData.Key() != "" {
  log.WriteString(":")
  log.WriteString(matchData.Key())
}
log.WriteString(".")
```